src/applications/mhv/medications/containers/LandingPage.jsx

```diff
@@ -1,23 +1,126 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { getPrescriptionsList } from '../actions/prescriptions';
 import MedicationsList from '../components/MedicationsList/MedicationsList';
 import MedicationsListSort from '../components/MedicationsList/MedicationsListSort';
+import { dateFormat, generateMedicationsPDF } from '../util/helpers';
 import PrintHeader from './PrintHeader';
 
 const LandingPage = () => {
+  const currentDate = new Date();
   const prescriptions = useSelector(
     state => state.rx.prescriptions.prescriptionsList,
   );
-  const dispatch = useDispatch();
+  const userName = useSelector(state => state.user.profile.userFullName);
+  const dob = useSelector(state => state.user.profile.dob);
 
+  const dispatch = useDispatch();
+  const [rxList, setRxList] = useState([]);
   useEffect(
     () => {
       dispatch(getPrescriptionsList());
     },
     [dispatch],
   );
 
+  useEffect(
+    () => {
+      let arr;
+      if (prescriptions) {
+        arr = prescriptions.map(rx => {
```

Comment:
What would you think of moving this logic to its own function, maybe named something like `buildPrescriptionList`? It would make it easy to see what's happening in the `useEffect` at a glance 🙂

Reply:
(This is a matter of taste, not a must at all!)

Reply:
I agree, i just pushed up this change.